_drafts/SC1-2-2-video-captions.md

```diff
@@ -0,0 +1,88 @@
+---
+name: video captions
+description: |
+  Captions are available for audio information in non-streaming `video` elements.
+
+success_criterion:
+- 1.2.2 # Captions (Prerecorded)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- [Audio output][]
+- [Visual output][]
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming][] `video` element [visible on the page][] where the video contains [audio][].
+
+### Expectation
+
+Of each test target, audio information that is not expressed visually in the video, is available through [captions][].
+
+*Note*: Captions can be either embedded in the video file itself or can be made available trough a separate track.
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- http://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-captions.html
+- https://www.w3.org/TR/WCAG20-TECHS/G93.html
+- https://www.w3.org/TR/WCAG20-TECHS/G87.html
+- https://www.w3.org/TR/WCAG20-TECHS/H95.html
+
+## Test Cases
+
+### Passed
+
+```html
+<video src="../test-assets/perspective-video/perspective-video-with-captions.mp4" controls></video>
+```
+
+```html
+<video src="../test-assets/perspective-video/perspective-video.mp4" controls>
+  <track src="/test-assets/perspective-video/perspective-caption.vtt" kind="captions">
+</video>
+```
+
+### Failed
+
+```html
+<video src="../test-assets/perspective-video/perspective-video.mp4" controls></video>
+```
+
+```html
+<video src="../test-assets/perspective-video/perspective-keyboard-compatibility-video.mp4" controls>
+  <track src="/test-assets/perspective-video/perspective-incorrect-caption.vtt" kind="captions">
+</video>
+```
+
+### Inapplicable
+
```

Comment:
Add a test case for when the audio information is available as text on the page and the text is associated with the video clip.

There should also be a test case with a live video source, but I have no idea how to ensure the resource would be available for testing purposes.

Reply:
We are adding a test case with "ariadescribedby". Thanks for the feedback.